applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -98,6 +98,12 @@ def __init__(self, model, custom_settings):
             self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DOMAIN_SIZE, domain_size)
             self.solver_imports_model_part = True
 
+        # TODO: improve robustness of the "if" adding element names which support OSS
```

Comment:
Please do this before merging. Otherwise the user will specify use OSS but will silently hide that the solution has nothing to do with OSS.